src/bin/cargo/commands/owner.rs

```diff
@@ -6,46 +6,137 @@ use cargo_credential::Secret;
 pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
-        .arg(Arg::new("crate").value_name("CRATE").action(ArgAction::Set))
+        .arg_quiet()
+        .arg(Arg::new("crate").hide(true))
+        .arg_required_else_help(true)
+        .args_conflicts_with_subcommands(true)
+        .override_usage(
+            "\
+       cargo owner add    <OWNER_NAME> [CRATE_NAME] [OPTIONS]
+       cargo owner remove <OWNER_NAME> [CRATE_NAME] [OPTIONS]
+       cargo owner list   [CRATE_NAME] [OPTIONS]",
+        )
+        // The following three parameters are planned to be replaced in the form of subcommands.
+        // refer to issue: https://github.com/rust-lang/cargo/issues/4352
         .arg(
             multi_opt(
                 "add",
                 "LOGIN",
                 "Name of a user or team to invite as an owner",
             )
-            .short('a'),
+            .short('a')
+            .hide(true),
         )
         .arg(
             multi_opt(
                 "remove",
                 "LOGIN",
                 "Name of a user or team to remove as an owner",
             )
-            .short('r'),
+            .short('r')
+            .hide(true),
         )
-        .arg(flag("list", "List owners of a crate").short('l'))
+        .arg(flag("list", "List owners of a crate").short('l').hide(true))
+        .subcommands([
+            for_subcommand_add_arg(
+                Command::new("add")
+                    .about("Name of a user or team to invite as an owner")
+                    .arg_quiet()
```

Comment:
Why is `arg_quiet` needed for these?  Isn't it a global?

Reply:
already edited.